src/msg_recorder/replayer.h

```diff
@@ -72,19 +72,21 @@ class MessageReplayer : public CRNamedNode<MessageReplayer> {
 };
 
 template<CRMessageType message_t>
-void MessageFromStr(message_t& msg, const std::string& serialized_msg);
-
-template<CRMessageType message_t>
-void MessageReplayer::RegisterChannel(const MessageReplayer::channel_subid_t subid) {
+bool MessageReplayer::RegisterChannel(const MessageReplayer::channel_subid_t subid) {
+    auto itr = GetRecordItr(GetTypeName<message_t>(), subid);
```

Comment:
iter recommended.

Reply:
I used `itr` everywhere in this project, so let's keep the name consistent here.